### DISCUSSION TITLE
Option to add SimpleIP to shell startup on boot or similar

### DISCUSSION THREAD

**Discussion by zapy85** [snappysnappydog/Simple-IP-Config#120] [Ideas]
As discussed in #52 we would like;

An option in the settings menu to start SimpleIP on boot/login without adding your own shortcut and all that.
And
An option in the settings menu to always start SimpleIP to systray.

This would allow us to have it start at boot/login to systray but would also leave us with an option to start it manually to systray with the second option checked, and if not it would start to front.

**Comment by zapy85** [comment]
Hi Kurtis!
Hows the development going on the new version? :-)

**Comment by controlzone** [comment]
This would be a great feature
